recipes/pyzmq/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
```

Comment:
Already conda-build's behaviour on Linux/OSX. On Windows it (the equivalent behaviour) isn't the case though.